Automattic/sensei
# Add Email settings form

Fixes #6447 

### Changes proposed in this Pull Request

* Add settings form.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );`
* Go to Sensei LMS -> Settings -> Emails -> Settings
* Try to change values and save. It is expected to see a notification about the successful saving on the same page.
* Try to leave fields blank. It is expected to see a notification about errors while saving on the same page.

### Screenshots
<img width="1679" alt="CleanShot 2023-02-13 at 23 27 48@2x" src="https://user-images.githubusercontent.com/329356/218519879-4fa26e26-794b-47f9-9821-31d88a444d2a.png">
<img width="1680" alt="CleanShot 2023-02-13 at 23 28 33@2x" src="https://user-images.githubusercontent.com/329356/218519909-0e825fd6-0cbb-4040-96f9-42657436cc9e.png">
<img width="1680" alt="CleanShot 2023-02-13 at 23 28 09@2x" src="https://user-images.githubusercontent.com/329356/218519923-b692cbc8-49f1-4648-acf2-63922dea151e.png">
